docs/decisions/0013-system-taxonomy-auto-tagging.rst

```diff
@@ -4,25 +4,23 @@
 Context
 --------
 
-One main characteristic of system-defined taxonomies is automatic content tagging. 
+One main characteristic of system-defined taxonomies is automatic content tagging.
 It is necessary to implement this functionality when the associated content is created or edited.
 
 Decision
 ---------
 
-Use `openedx-filters`_ to call the auto tagging function after content creation/edition.
-
-Filters
+Events
 ~~~~~~~~
 
-It is necessary to create `filters`_ for each content for creation/edition: ``CourseCreation``, ``LibraryCreation``, etc.
-This filters will live on ``openedx-filters``.
+It is necessary to create `events`_ for each content for creation/edition: ``CourseCreation``, ``LibraryCreation``, etc.
+This events will live on ``openedx-events``.
```

Comment:
Could you add a link here, and expand it [below](https://github.com/openedx/openedx-learning/pull/61/files#diff-2113051e4c045e58c5e9eed6c560e78598409cca5f91a830e601703a1256b5b3R45)?
```suggestion
This events will live in `openedx-events`_.
```